### PR TITLE
Feature: added max heap

### PR DESCRIPTION
**Pull request by manuelaguirre**

### CODE REVIEW

File: src/components/MainMenu.vue [hunk 1]
```diff
@@ -28,11 +28,17 @@ export default {
...
-      }, {
+      },
+      {
         name: 'Min Heap',
         imageUrl: 'https://cdn.cienradios.com/wp-content/uploads/sites/4/2020/02/pablito-lescano.gif',
         code: 'min-heap',
       },
+      {
+        name: 'Max Heap',
+        imageUrl: 'https://cdn.cienradios.com/wp-content/uploads/sites/4/2020/02/pablito-lescano.gif',
```
Comment (sebastianfdez) [inline src/components/MainMenu.vue:39]: In #41 I added real images related with the structure. Try to find one appropiate so we can have a nice main menu :)

File: src/components/MaxHeapVisualizer.vue [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        Structure: Max Heap
+      </h1>
+    </div>
+    <div class="flex buttons">
+      <InsertInput
+        :disabled="isDisabled"
+        @myEvent="insertInputEvent"
+      />
+      <DeleteInput
+        :onlypop="onlypop"
```
Comment (sebastianfdez) [inline src/components/MaxHeapVisualizer.vue:23]: Maybe we can change the text in the DeleteInput component to Pop, when onlypop = true

File: src/components/MainMenu.vue (hunk 1 above)
Comment (manuelaguirre) [reply src/components/MainMenu.vue:39]: Ok, gonna look for some nice max heap img

File: src/components/MaxHeapVisualizer.vue (hunk 2 above)
Comment (manuelaguirre) [reply src/components/MaxHeapVisualizer.vue:23]: I don't really understand what this flag is for

Comment (sebastianfdez) [reply src/components/MaxHeapVisualizer.vue:23]: Heap don't use classical delete(key) function. They only pop the node with the higher (or lower) priority. So te hide the text input in the DeleteInputComponent, we use that flag

Comment (manuelaguirre) [reply src/components/MaxHeapVisualizer.vue:23]: Ah ok, I think the current UI is sophisticated and lets the user know that they don't have choice of element when they are removing from the heap. But if you feel like it, go ahead.

**Pull request merged**